hack/test-e2e-local.sh

```diff
@@ -52,6 +52,7 @@ if [[ "$1" != "operator" ]]; then
     e2e-wake-up-wl.local
     e2e-migrate.local
     e2e-migrate-wl.local
+    e2e-mgr-hib.local
```

Comment:
Please update
https://github.com/gardener/gardener/blob/4a7ec00a52c9af5a995e244d1d073bd12a3e89ab/docs/deployment/getting_started_locally.md#L226
and https://github.com/gardener/gardener/blob/f1b225aeede579bc733739b90b0186609eec033f/docs/deployment/content/remote-local-setup.yaml#L36 as well.